examples/sanic/main.py

```diff
@@ -0,0 +1,124 @@
+from contextvars import ContextVar
+from datetime import *
+
+import requests
+from pybrake.middleware.sanic import init_app
+from sanic import Sanic
+from sanic import response
+from sqlalchemy import INTEGER, Column, String, select, insert
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.ext.asyncio import create_async_engine
+from sqlalchemy.orm import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+app = Sanic(__name__)
+
+bind = create_async_engine("sqlite+aiosqlite:///./test.db", echo=True)
+Base = declarative_base()
+
+app.config["PYBRAKE"] = dict(
+    project_id=9999,
+    project_key="xxxxxxxxxxxxxxxxxxxxxxxxx",
+    environment="test",
+    performance_stats=True,  # False to disable APM
```

Comment:
```suggestion
    performance_stats=True, # False to disable APM
```

Reply:
According to pylint, two spaces are required.